js/components/controlBar.js

```diff
@@ -550,284 +550,208 @@ var ControlBar = React.createClass({
+          <AccessibleButton
+            className="oo-mute-unmute oo-control-bar-item"
+            onClick={this.handleVolumeIconClick}
+            onMouseOver={this.highlight}
+            onMouseOut={this.removeHighlight}
+            focusId={CONSTANTS.FOCUS_IDS.MUTE_UNMUTE}
+            ariaLabel={volumeAriaLabel}>
+            <Icon
+              {...this.props}
+              icon={volumeIcon}
+              ref="volumeIcon"
+              style={this.props.skinConfig.controlBar.iconStyle.inactive} />
+            <Tooltip
+              {...tooltipConfig}
+              parentKey={CONSTANTS.CONTROL_BAR_KEYS.VOLUME}
+              text={mutedInUi ? CONSTANTS.SKIN_TEXT.UNMUTE : CONSTANTS.SKIN_TEXT.MUTE} />
+          </AccessibleButton>
+          <VolumeControls {...this.props} />
+        </div>
+      ),
+
+      timeDuration: (
+        <a
+          key={CONSTANTS.CONTROL_BAR_KEYS.TIME_DURATION}
+          className="oo-time-duration oo-control-bar-duration"
+          style={durationSetting}>
+          <span>{playheadTimeContent}</span>{totalTimeContent}
+        </a>
+      ),
+
+      flexibleSpace: (
+        <div key={CONSTANTS.CONTROL_BAR_KEYS.FLEXIBLE_SPACE} className="oo-flexible-space oo-control-bar-flex-space" />
+      ),
+
+      moreOptions: (
+        <a
+          key={CONSTANTS.CONTROL_BAR_KEYS.MORE_OPTIONS}
+          className="oo-more-options oo-control-bar-item"
+          onClick={this.handleMoreOptionsClick}
+          aria-hidden="true">
+          <Icon
+            {...this.props}
+            icon="ellipsis"
+            style={dynamicStyles.iconCharacter}
+            onMouseOver={this.highlight}
+            onMouseOut={this.removeHighlight} />
+          <Tooltip
+            {...tooltipConfig}
+            parentKey={CONSTANTS.CONTROL_BAR_KEYS.MORE_OPTIONS}
+            text={CONSTANTS.SKIN_TEXT.MORE_OPTIONS} />
+        </a>
+      ),
+
+      quality: (
+        <div key={CONSTANTS.CONTROL_BAR_KEYS.QUALITY} className="oo-popover-button-container">
+          <AccessibleButton
+            ref={function(e) {
+              this.setToggleButtons(CONSTANTS.MENU_OPTIONS.VIDEO_QUALITY, e);
+            }.bind(this)}
+            style={selectedStyle}
+            className={qualityClass}
+            focusId={CONSTANTS.FOCUS_IDS.VIDEO_QUALITY}
+            ariaLabel={CONSTANTS.ARIA_LABELS.VIDEO_QUALITY}
+            ariaHasPopup={true}
+            ariaExpanded={this.props.controller.state.videoQualityOptions.showPopover ? true : null}
+            onClick={this.handleQualityClick}>
             <Icon
               {...this.props}
-              icon="ellipsis"
+              icon="quality"
               style={dynamicStyles.iconCharacter}
               onMouseOver={this.highlight}
-              onMouseOut={this.removeHighlight}
-            />
+              onMouseOut={this.removeHighlight} />
             <Tooltip
```

Comment:
What happened with the tooltip for flexible space? I'm assuming it was removed because it wasn't needed?